dspace-server-webapp/src/test/java/org/dspace/app/rest/EPersonRestRepositoryIT.java

```diff
@@ -110,6 +135,34 @@ public void createTest() throws Exception {
         // TODO cleanup the context!!!
     }
 
+    @Test
+    public void createAnonAccessDeniedTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+        // we should check how to get it from Spring
+        ObjectMapper mapper = new ObjectMapper();
+        EPersonRest data = new EPersonRest();
+        EPersonRest dataFull = new EPersonRest();
+        MetadataRest metadataRest = new MetadataRest();
+        data.setEmail("createtest@fake-email.com");
+        data.setCanLogIn(true);
+        MetadataValueRest surname = new MetadataValueRest();
+        surname.setValue("Doe");
+        metadataRest.put("eperson.lastname", surname);
+        MetadataValueRest firstname = new MetadataValueRest();
+        firstname.setValue("John");
+        metadataRest.put("eperson.firstname", firstname);
+        data.setMetadata(metadataRest);
+        dataFull.setEmail("createtestFull@fake-email.com");
+        dataFull.setCanLogIn(true);
+        dataFull.setMetadata(metadataRest);
+
+        getClient().perform(post("/api/eperson/epersons")
+                                         .content(mapper.writeValueAsBytes(data))
+                                         .contentType(contentType)
+                                         .param("projection", "full"))
+                            .andExpect(status().isUnauthorized());
```

Comment:
After this POST, we should have a test to verify that the person was *not* created.

Reply:
This is now verified